pebbles/models.py

```diff
@@ -489,6 +489,20 @@ def generate_name(prefix):
         return '%s%s-the-%s' % (prefix, names.get_first_name().lower(), random.choice(NAME_ADJECTIVES))
 
 
+class InstanceLog(db.Model):
+    __tablename__ = 'instance_logs'
+    id = db.Column(db.String(32), primary_key=True)
+    instance_id = db.Column(db.String(32), db.ForeignKey('instances.id'), index=True, unique=False)
+    log_level = db.Column(db.String(8))
+    log_type = db.Column(db.String(64))
+    timestamp = db.Column(db.Float)
+    message = db.Column(db.String(256))
```

Comment:
I would say this again, these the logs for instances, i.e. the creators of the driver are sending those messages manually. The full length logs are kept in the backend and these are supposed to be a high level thing. I don't think it would cross the stated character limit. Anyway, clamping would cause un-necessary readability issues. So, I can just change it to Text (for worst case scenario).